docs/source/perf_diff.rst

```diff
@@ -1,6 +1,21 @@
 Performing Differentiation
 ==========================
 
+Restrictions on the target function
+-----------------------------------
+
+ForwardDiff.jl can only differentiate functions that adhere to the following rules:
+
+- **The function must be univariate** (an exception to this is found in the ``jacobian`` method below).
```

Comment:
Univariate usually means scalar-valued input. Taking only one argument (which itself could be a vector) is something else, I think.